### PR TITLE
Lions schema

### PR DESCRIPTION
**Pull request by redyaffle**
@bantic @iezer For either of your review.

---

This builds out a preliminary schema for users, lions, images, image_sets, and organizations, as well as associations and validations between these models. Some tweaking will be needed later on in order to expand the metadata per Justin's feedback, as well as some work to specify age fields correctly. But this gets us started and I'll continue expanding from here. 

I also expanded ActiveAdmin to create organizations and users. You can now make yourself an ActiveAdmin user, log in with your account, and use that account to create organizations and users as needed.

### CODE REVIEW

File: app/models/image_type.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class ImageType < ActiveRecord::Base
+  has_many :images
+
+  validates :name,
+    :inclusion  => { :in => [ 'cv', 'full-body', 'whisker', 'main-id' ],
+    :message    => "%{value} is not a valid relationship status" }
```
Comment (bantic) [inline app/models/image_type.rb]: s/relationship status/image type/

**Comment by bantic** [comment]
looks good — I'd like to see some basic unit tests but otherwise looks good.

File: app/models/image.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class Image < ActiveRecord::Base
+  belongs_to :image_set
+  belongs_to :image_type
+end
```
Comment (iezer) [inline app/models/image.rb]: @redyaffle  I'd suggest validating that the image has a URL. Did you see the `s3-upload-rebased` branch I created? I also created an image model there and started working on the API. We can merge this branch first and I'll remove some stuff from that branch later, but I just wanted to suggest that we port over that [valdation](https://github.com/201-created/lion-guardians-api/compare/s3-upload-rebased#diff-69e1773ce93d7ede3d3d0c7f4e6c7024R2) and also the [unit tests](https://github.com/201-created/lion-guardians-api/compare/s3-upload-rebased#diff-31fdbb7677a4de0d6fbce02d2f1120c7R1).

Comment (redyaffle) [inline app/models/image.rb]: @iezer I'll definitely port those over. I didn't realize I was duplicating efforts there - sorry about that.  I'll take a look through the s3-upload-rebased branch and bring over the validation. I think I'll leave the unit tests on your branch as I don't have the Fabricator infrastructure, but I'm happy to update your branch with this schema if that'd be helpful.

**Comment by redyaffle** [comment]
I'll test the validations once we also have @iezer's work with fabricator merged as well. I made an additional Trello card to do so.

**Pull request merged**